laravel-app/app/Interfaces/PostServiceInterface.php

```diff
@@ -12,4 +12,15 @@ interface PostServiceInterface
      * @param array<mixed, mixed> $validatedData
      */
     public function createPost(User $user, array $validatedData): UserPost;
+
+    /**
+     * To update an existing post.
+     *
+     * @param UserPost $post
+     * @param array<string, mixed> $validatedData
+     * @return bool
```

Comment:
Remove docblock return and params!!!

Reply:
Yes sir, I have removed the return and params but the larastan check return an error requiring us the params to be listed.  So I have removed the return and param of UserPost but retained the param for validatedData. Thank you ❤️